app/src/main/java/com/soen490chrysalis/papilio/view/ActivitiesFragment.kt

```diff
@@ -38,7 +40,14 @@ class ActivitiesFragment : Fragment()
     ) : View?
     {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_activities, container, false)
+        val view = inflater.inflate(R.layout.fragment_activities, container, false)
```

Comment:
Use view binding. Here is the documentation on how to do it: [https://developer.android.com/topic/libraries/view-binding#fragments](url)